x-pack/test/apm_api_integration/common/registry.ts

```diff
@@ -17,6 +17,7 @@ type ArchiveName =
   | 'apm_8.0.0'
   | '8.0.0'
   | 'metrics_8.0.0'
+  | 'metrics_and_apm'
```

Comment:
Do you mean "infra_metrics" or a type of apm metric (system, transaction etc?) We should clarify what type of metric is meant.

Reply:
ah! I refer to infra metrics. I'll update it to `infra_metrics_and_apm`

Reply:
fixed: 091386756e28a584d1eccc24330004b6771b5176